com.github.ransome1.sleek.yml

```diff
@@ -5,6 +5,7 @@ sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
 base-version: '23.08'
 command: sleek
+name: sleek
```

Comment:
That's not a valid property, I think. I get a warning about that. Can be removed.